src/components/hosts/HostRowDetail.tsx

```diff
@@ -86,7 +86,7 @@ const DisksTable: React.FC<DisksTableProps> = ({ cluster, host, disks, installat
         },
         { title: <DiskLimitations disk={disk} /> },
         disk.driveType,
-        filesize(disk.sizeBytes || 0, 2, 'iec'),
+        filesize(disk.sizeBytes || 0, 2, 'si').replace(/b$/, 'B'),
```

Comment:
nit: can we have a new function in utils for that? To be sure we use that conversion consistently even in the future.

Reply:
sure I'll do that

Reply:
solved at c82add81f8f5b6f30515be9fba07eb2c8c31fda9